internal/plugin/plugin.go

```diff
@@ -95,6 +95,10 @@ func (p *Plugin) Exec() error {
 		options = append(options, cache.WithFallbackGenerator(keygen.NewStatic(p.Metadata.Commit.Branch)))
 	}
 
+	if p.Config.Override {
+		options = append(options, cache.WithOverride())
```

Comment:
As I commented above, just pass to value to options.
Let's make changing override value to false explicit.